stop_controlled_intersection_tactical_plugin/src/stop_controlled_intersection_tactical_plugin.cpp

```diff
@@ -100,7 +100,11 @@ bool StopControlledIntersectionTacticalPlugin::plan_trajectory_cb(cav_srvs::Plan
     ROS_DEBUG_STREAM("Current_downtrack"<< current_downtrack);
 
     std::vector<PointSpeedPair> points_and_target_speeds = maneuvers_to_points( maneuver_plan, wm_, req.vehicle_state);
-
+    ROS_DEBUG_STREAM("Maneuver to points size:"<< points_and_target_speeds.size());
+    ROS_DEBUG_STREAM("Printing points: ");
+    for(size_t i = 0;i<points_and_target_speeds.size();i++){
+        std::cout<<"Point "<<i<<", Speed: "<<points_and_target_speeds[i].speed<<std::endl;
```

Comment:
Replace with ros logging call

Reply:
This print can be removed. It was just being used as get the speed profile while unit testing. Since the points are being post processed, it doesn't give the final values used for trajectory generation.